api/v1beta1/aerospikecluster_types.go

```diff
@@ -183,11 +199,8 @@ func (c *AerospikeOperatorClientCertSpec) validate() error {
 		)
 	}
 
-	if c.TLSClientName != "" && !c.IsClientCertConfigured() {
-		return fmt.Errorf(
-			"tlsClientName is provided but client certificate is not: secretCertSource=%+v, certPathInOperator=%v+v",
-			c.SecretCertSource, c.CertPathInOperator,
-		)
+	if !c.IsClientCertConfigured() {
```

Comment:
checking TLSClientName was useless here, because before calling validate(), we were returning if TLSClientName is empty.
`if clientCertSpec.TLSClientName == "" {
			return fmt.Errorf("operator TLSClientName is not specified")
		}

		if err := clientCertSpec.validate(); err != nil {
			return err
		}`